logger/module.go

```diff
@@ -0,0 +1,90 @@
+package logger
+
+import (
+	"os"
+
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/rs/zerolog"
+)
+
+type loggerModule struct {
+	bus    modules.Bus
+	logger modules.Logger
+	config modules.LoggerConfig
+}
+
+// All loggers branch out of mainLogger, that way configuration changes to mainLogger propagate to others.
+var mainLogger = zerolog.New(os.Stderr).With().Timestamp().Logger()
+
+// The idea is to create a logger for each module, so that we can easily filter logs by module.
+// But we also need a global logger, because sometimes we need to log outside of modules, e.g. when the process just
+// started, and modules are not initiated yet.
+var Global = new(loggerModule).CreateLoggerForModule("global")
+
+var _ modules.LoggerModule = &loggerModule{}
+
+const (
+	ModuleName = "logger"
+)
+
+func Create(runtimeMgr modules.RuntimeMgr) (modules.Module, error) {
+	return new(loggerModule).Create(runtimeMgr)
+}
+
+func (*loggerModule) CreateLoggerForModule(moduleName string) modules.Logger {
+	return mainLogger.With().Str("module", moduleName).Logger()
+}
+
+func (*loggerModule) Create(runtimeMgr modules.RuntimeMgr) (modules.Module, error) {
```

Comment:
When do we use this Create versus the one long line 30?

Reply:
`logger.Create` is used in https://github.com/pokt-network/pocket/pull/301/commits/ea447886932c4711de5ebba42af308f073858ccf#diff-bcff8b319bd7c6f37325379309e118b3310f7308755b8ba7fb6a5de101c4d9edR76 - just pushed this.

Reply:
Discussed offline. We are following the pattern elsewhere in the codebase. Resolving.